processor/processor.go

```diff
@@ -805,6 +806,7 @@ func (proc *Handle) loadConfig() {
 	proc.config.enablePipelining = config.GetBoolVar(true, "Processor.enablePipelining")
 	proc.config.pipelineBufferedItems = config.GetIntVar(0, 1, "Processor.pipelineBufferedItems")
 	proc.config.subJobSize = config.GetIntVar(defaultSubJobSize, 1, "Processor.subJobSize")
+	proc.config.numPartitions = config.GetIntVar(1, 1, "Processor.numPartitions")
```

Comment:
We might need to reconsider and potentially remove this piece of code, since it might be too defensive, wdyt?
```go
	// If isolation mode is not none, we need to reduce the values for some of the config variables to more sensible defaults
	if proc.config.isolationMode != isolation.ModeNone {
		defaultSubJobSize = 400
		defaultMaxEventsToProcess = 2000
		defaultPayloadLimit = 20 * bytesize.MB
	}
```

Reply:
We should reconsider everything with respect to payload sizes and adaptive payload limiter since we want to rely on VPA to provide more memory and better throughput than restrict the throughput and keep the system safe. Thats a completely different discussion I guess